airflow/models/xcom.py

```diff
@@ -145,13 +150,16 @@ def get_many(
         dag_ids: Optional[Union[str, Iterable[str]]] = None,
         include_prior_dates: bool = False,
         limit: Optional[int] = None,
+        run_id: Optional[str] = None,
         session: Session = None,
     ) -> Query:
         """
         Composes a query to get one or more values from the xcom table.
 
         :param execution_date: Execution date for the task
         :type execution_date: pendulum.datetime
+        :param run_id: DagRun ID for the task
+        :type run_id: str
```

Comment:
Should we execution_date mark as a optional and add explicitly validation for required parameter - run_id or execution_date?

Reply:
I think the run_id and execution_date parameters should be mutually exclusive.  WDYT?